src/include/daos/tse.h

```diff
@@ -15,18 +15,24 @@
 
 #include <gurt/list.h>
 
-/* tse_task arguments max length (pthread_mutex_t is of different size between x86 and aarch64). */
+#include <pthread.h>
+#include <inttypes.h>
+#include <stdbool.h>
+
+/** tse_task arguments max length (pthread_mutex_t is of different size between x86 and aarch64). */
 #define TSE_TASK_ARG_LEN	(840 + sizeof(pthread_mutex_t))
-/* internal tse private data size (struct tse_task_private) */
+/** internal tse private data size (struct tse_task_private) */
 #define TSE_PRIV_SIZE		(TSE_TASK_ARG_LEN + 136)
-/* tse_task is used to track single asynchronous operation (8 bytes used for public members). */
+/** tse_task is used to track single asynchronous operation (8 bytes used for public members). */
 #define TSE_TASK_SIZE		(TSE_PRIV_SIZE + 8)
 
+/** A single task */
```

Comment:
that sounds like a useless comment.

Reply:
Our documentation requires all structures and entries to be documented, some text is needed here, feel free to suggest something better.

Reply:
"struct for a task object that is used for tracking an operation"